src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx

```diff
@@ -229,6 +229,13 @@ const MessageDetails: FC<MessageDetailsProps> = ({
 
   const onParticipantClick = (userEntity: User) => togglePanel(PanelState.GROUP_PARTICIPANT_USER, userEntity);
 
+  function capitalizeFirstLetter(emojiName: string) {
+    if (emojiName && emojiName.trim() !== '') {
+      return emojiName.charAt(0).toUpperCase() + emojiName.slice(1);
```

Comment:
We have `capitalizeFirstChar` method in our `StringUtil.ts` let's use it

Reply:
I want to perform the sanity check.

if (emojiName && emojiName.trim() !== '') 

shall i add it to the capitalizeFirstChar method?

Reply:
No, we can do just `return capitalizeFirstChar(emojiName)` after sanity check is valid - we will need to rename the original method then.

Reply:
why not adding it to the capitalizeFirstChar method? this is a basic sanity check, it should be there in the method.